src/composables/useNfts.ts

```diff
@@ -0,0 +1,46 @@
+import { ref, computed, Ref } from 'vue';
+import snapshot from '@snapshot-labs/snapshot.js';
+
+const nfts: Ref<any[]> = ref([]);
+const loading = ref(true);
+const loaded = ref(false);
+
+export function useNfts() {
+  async function loadNfts(address) {
+    loading.value = true;
+    const key = 'ckey_2d082caf47f04a46947f4f212a8';
+    const url = `https://api.covalenthq.com/v1/1/address/${address}/balances_v2/?quote-currency=USD&format=JSON&nft=true&key=${key}`;
+    const results = await snapshot.utils.getJSON(url);
+    nfts.value = results.data.items
+      .filter(item => item.type === 'nft' && item.nft_data)
+      .map(item => {
+        const id = item.nft_data[0]?.token_id;
+        const title =
+          item.nft_data[0]?.external_data?.name ??
+          item.contract_name ??
+          'Untitled';
+        const displayTitle =
+          title.match(/(#[0-9]+)$/) || !id ? title : `${title} #${id}`;
+
+        const image =
+          item?.nft_data[0]?.external_data.image ??
+          `https://stamp.fyi/token/${item.contract_address}?s=256`;
```

Comment:
Would be nice to have Stamp support for NFTs.

Reply:
Ok yes, feel free to add an issue for this, we could add support with url like this `stamp.fyi/nft/$address/$tokenId`. Also we should always use the cdn subdomain `cdn.stamp.fyi` to display image, at some point we will have a landing page at stamp.fyi. But currently I dont think we should use stamp here with the token url.

Reply:
Unless you are using it to get Blockie fallback image?

Reply:
Here it's only to load the fallback (blockie) image in two cases:
1. image is missing on the NFT
2. image is there, but it fails to load ([here](https://github.com/snapshot-labs/sx-ui/blob/9809780a3f8a08381f8f10716e6bc169ee1abb28/src/components/NftPreview.vue#L7-L9)).